Client/Frontend/Strings.swift

```diff
@@ -1415,3 +1415,57 @@ extension String {
     public static let ProtectionStatusSecure = MZLocalizedString("ProtectionStatus.Secure", value: "Connection is secure", comment: "This is the value for a label that indicates if a user is on a secure https connection.")
     public static let ProtectionStatusNotSecure = MZLocalizedString("ProtectionStatus.NotSecure", value: "Connection is not secure", comment: "This is the value for a label that indicates if a user is on an unencrypted website.")
 }
+
+// MARK: - v39 Strings
+extension String {
+    //Customizing strings for the Start at Home feature is settings
+    public static let SettingsCustomizeHomeStartAtHomeSectionTitle = MZLocalizedString("Settings.Home.Option.StartAtHome.Title", value: "Opening screen", comment: "Option in settings to set start at home feature timing")
```

Comment:
`"Option in settings to set start at home feature timing"` 

nit: I think if we add `Title for section in settings to configure opening home screen` would be more clear to translators